README.md

```diff
@@ -57,50 +57,71 @@ If you're using an M1 or M2 Mac, you may have trouble installing the `grpc` gem;
 ### Usage
 
 ```ruby
+# An example of the basic functionality of
+# Momento::SimpleCacheClient.
+
 require 'momento'
 
 # Get your Momento token from an environment variable.
-token = ENV.fetch('MOMENTO_AUTH_TOKEN')
+TOKEN = ENV.fetch('MOMENTO_AUTH_TOKEN')
 
 # Cached items will be deleted after 12.5 seconds.
-ttl = 12.5
+TTL = 12.5
+
+# The name of the cache to create *and delete*
+CACHE_NAME = ENV.fetch('MOMENTO_CACHE_NAME')
 
 # Instantiate a Momento client.
 client = Momento::SimpleCacheClient.new(
-  auth_token: token,
-  default_ttl: ttl
+  auth_token: TOKEN,
+  default_ttl: TTL
```

Comment:
here as well? `deafult_ttl_seconds`
In PHP, we have it like [this](https://github.com/momentohq/client-sdk-php/blob/main/src/Cache/SimpleCacheClient.php#L54).

Reply:
In the future, the plan is to take a time interval object as well as integer seconds. I'd like to avoid having mutually exclusive `default_ttl_seconds` and `default_ttl_interval` parameters.

Reply:
Oh like `TimeSpan` in .NET SDK! 
Got it 👍

Reply:
Yeah, except Ruby doesn't have one built in. 😞 We're leaving it until later.